Java/syntax_test_java.java

```diff
@@ -1,6 +1,14 @@
 // SYNTAX TEST "Java.sublime-syntax"
 
 public class SyntaxTest {
+    private String memberString = "Hello";
+    private String memberString2 = new String("Hello");
+    private String memberString3 = String.valueOf("Hello");
```

Comment:
done. It's pointing at the start of the string literal value itself. When it was pointing at the actual starting double-quote mark, one character back, it still passed the test; presumably because the string.quoted.double.java context is in force even though it's not the last scope in the list, that being, correctly I'm sure, `punctuation.definition.string.begin.java`.

(edit: add this as a line comment instead)